third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -535,12 +554,12 @@ int WinUWPH264EncoderImpl::SetRates(
     }
 
     EncodedImageCallback* tempCallback = encodedCompleteCallback_;
-    Release();
+    ReleaseWriter();
     {
       rtc::CritScope lock(&callbackCrit_);
       encodedCompleteCallback_ = tempCallback;
     }
-    InitEncoderWithSettings(&codec_);
+    InitEncoderWithSettings();
```

Comment:
Can we remove the discrepancy between "Init**Encoder**WithSettings" and "Release**Writer**"? I believe those two functions work in pair, so should have similar names, no? Can we just rename to `InitWriter()` instead?